src/commands/context/inspectDockerContext.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext, openReadOnlyContent } from 'vscode-azureextensionui';
+import { localize } from '../../localize';
+import { dockerContextManager } from '../../utils/dockerContextManager';
+import { selectDockerContext } from './selectDockerContext';
+
+export async function inspectDockerContext(_context: IActionContext): Promise<void> {
```

Comment:
Here and in the rest of the commands, I'd rename `_context` / `context` to `actionContext`, to avoid confusion.

Reply:
Yes, absolutely. Fixed